src/stories/form-ui.tsx

```diff
@@ -0,0 +1,94 @@
+import { FormEventHandler, useState } from "react";
+
+export default function FormUI() {
+    const [passwordError, setPasswordError] = useState<string | undefined>();
```

Comment:
There is no need to add ` | undefined` to the types. `undefined` will always be added automatically to a state's type when there is no default value defined.